File: Pertemuan_5/predikatKelulusan.py
# Deklarasikan Variable IPK untuk memberi input
ipk = input("Masukan IPK Anda = ")
ipk = float(ipk)

# Deklarasikan Variable waktuStudi untuk memberi input
waktuStudi = input("Masukan Waktu Studi = ")
waktuStudi = float(waktuStudi)

# Program untuk menyatakan sebuah konsidi yang akan di jalankan
if   (ipk >= 3.51 and ipk <=4 and waktuStudi <=4):
    print("Selamat! Anda Meraih Predikat Summa Cumlaude")
elif (ipk >= 3.51 and ipk <=4 and waktuStudi > 4):
    print("Selamat! Anda Meraih Predikat Cumlaude")
elif (ipk >= 3.01 and ipk < 3.51):
    print("IPK Anda Sangat Memuaskan")
elif (ipk >= 2.76 and ipk < 3.01):
    print("IPK Anda Memuaskan")
elif (ipk >= 2.00 and ipk < 2.76):
    print("IPK Anda Cukup")
elif (ipk < 2.00):
    print("Anda Tidak Lulus")
else:
    print("IPK Tidak Valid")

File: Pertemuan_5/predikatKelulusan.java
package Pertemuan_5;

// Panggil Function Input
import java.util.Scanner;

public class predikatKelulusan {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // deklarasikan Variable input ipk
        System.out.print("Masukan IPK Anda = ");
        double ipk = scanner.nextDouble();

        // deklarasikan Variable input waktuStudi
        System.out.print("Masukan waktu studi = ");
        double waktuStudi = scanner.nextDouble();

        // Program untuk menyatakan sebuah konsidi yang akan di jalankan
        if (ipk >= 3.51 && ipk <= 4 && waktuStudi <= 4) {
            System.out.println("Selamat Anda Meraih Predikat Summa Cumlaude");
        }else if (ipk >= 3.51 && ipk <= 4 && waktuStudi > 4)  {
            System.out.println("Selamat Anda Meraih Predikat Cumlaude");
        }else if (ipk >= 3.01 && ipk < 3.51)  {
            System.out.println("IPK Anda Sangat Memuaskan");
        }else if (ipk >= 2.76 && ipk < 3.01)  {
            System.out.println("IPK Anda Memuaskan");
        }else if (ipk >= 2.00 && ipk < 2.76 ) {
            System.out.println("IPK Anda Cukup");   
        }else if (ipk < 2.00) {
            System.out.println("Anda Tidak Lulus");
        }else {
            System.out.println("Tidak Valid");
        }
    }
}
    
